_data/sites.json

```diff
@@ -5656,6 +5656,20 @@
             "linktr.ee"
         ]
     },
+    
+    {
+        "name": "Terms of Service; Didn't Read",
```

Comment:
We keep the JSON ordered by the service names, so this needs to come a lot later down the file, can you please fix the ordering? Thanks!